docs/src/catalyst-standards/draft-cips/x509-role-registration-metadata/x509-roles.cddl

```diff
@@ -0,0 +1,123 @@
+; An envelope for cardano metadata which holds the x509 metadata.
+; We do thsi so that the underlying metadata is not restircted to
+; cardano ledger implementation.
```

Comment:
```suggestion
; An envelope for Cardano metadata which holds the x509 metadata.
; We do this so that the underlying metadata is not restircted to
; Cardano ledger implementation.
```